cluster/manifests/kube-cluster-autoscaler/daemonset.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
   namespace: kube-system
   labels:
     application: kube-cluster-autoscaler
-    version: {{.Cluster.ConfigItems.cluster_autoscaler_version}}
```

Comment:
afaik, the version label is something we ought to set.

Reply:
Is it okay if I put it back when we roll it out to all clusters? I could also do a follow-up PR if you'd prefer it the other way around.

Reply:
Sure, that's fine.